.github/workflows/current-workflow.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Check out repository code
         uses: actions/checkout@v4
       - run: echo "The repository ${{ github.repository }} has been cloned to runner."
-      - run: echo "Workflow is now ready to test code on runner."
+      - run: echo "Workflow is now ready to tests code on runner."
```

Comment:
tests -> test

Reply:
that red mean it's been changed to the green, i believe we wanted the the test folder to be called tests